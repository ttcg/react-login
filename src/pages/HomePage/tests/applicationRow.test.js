import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ApplicationRow from '../applicationRow';

it('should renders ApplicationRow', () => {

    const data =
    {
        "applicationName": "Test App",
        "applicationTypeId": 999,
    };

    const component = shallow(<ApplicationRow data={data} />);

    expect(component.text()).toContain(data.applicationName);
    expect(toJson(component)).toMatchSnapshot();
});