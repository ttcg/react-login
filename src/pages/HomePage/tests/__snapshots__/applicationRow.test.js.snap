// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders ApplicationRow 1`] = `
<div
  className="wrapper"
  onClick={[Function]}
>
  <div
    className="image-wrapper"
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Object {
          "icon": Array [
            512,
            512,
            Array [],
            "f07b",
            "M464 128H272l-54.63-54.63c-6-6-14.14-9.37-22.63-9.37H48C21.49 64 0 85.49 0 112v288c0 26.51 21.49 48 48 48h416c26.51 0 48-21.49 48-48V176c0-26.51-21.49-48-48-48zm0 272H48V112h140.12l54.63 54.63c6 6 14.14 9.37 22.63 9.37H464v224z",
          ],
          "iconName": "folder",
          "prefix": "far",
        }
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size="3x"
      spin={false}
      symbol={false}
      title=""
      transform={null}
    />
  </div>
  <div
    className="text-wrapper"
  >
    Test App
    <span
      className="applicationText"
    >
      <ApplicationName
        id={999}
      />
    </span>
  </div>
</div>
`;
